mocks/mocks.go

```diff
@@ -577,7 +453,40 @@ func (sa *StorageAuthority) CountInvalidAuthorizations2(ctx context.Context, req
 }
 
 func (sa *StorageAuthority) GetValidAuthorizations2(ctx context.Context, req *sapb.GetValidAuthorizationsRequest) (*sapb.Authorizations, error) {
-	return nil, nil
+	if *req.RegistrationID == 1 || *req.RegistrationID == 5 || *req.RegistrationID == 4 {
```

Comment:
I think this function would be a little bit less triangle shaped if you inverted this `if` statement and returned early when it was true, knocking out one level of indentation over the body. E.g.:
```
if *req.RegistrationID != 1 && *req.RegistrationID != 5 && *req.RegistrationID != 4 {
  return &sapb.Authorizations{}, nil
}
....
```